dts/bindings/gpio/litex,gpio.yaml

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2019 Antmicro <www.antmicro.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+title: Litex GPIO
+
+description: >
+    This is a representation of the Litex GPIO nodes
+
+compatible: "litex,gpio"
+
+include: [gpio-controller.yaml, base.yaml]
+
+properties:
+    port-is-output:
+      type: boolean
+      description: Indicates if the port is an output port
+
+    reg:
+      required: true
+
+    label:
+      required: true
+
+    ngpios:
+      type: int
```

Comment:
remove `type` and `description`, should be inherited from gpio-controller.yaml

Reply:
removed